src/Evolution/Systems/RelativisticEuler/Valencia/Tags.hpp

```diff
@@ -13,20 +13,26 @@ class DataVector;
 
 namespace RelativisticEuler {
```

Comment:
Could you make the commit message more informative? For example, why do you need this change? I think none of the tags have been templated on the data type so far and that's been fine. Why do you need this?

Reply:
I thought we wanted analytic solutions to be templated on the data type. That's what I was asked to do for `NewtonianEuler`'s `IsentropicVortex`. I don't remember the reason off the top of my head now.